src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -1040,7 +1076,7 @@ assign(
 
 ReactDOMComponent.injection = {
   injectIDOperations: function(IDOperations) {
-    ReactDOMComponent.BackendIDOperations = BackendIDOperations = IDOperations;
+    ReactDOMComponent.BackendIDOperations = IDOperations;
```

Comment:
ReactDOMTextArea is the only thing left using `ReactDOMComponent.BackendIDOperations` - should we update that while we're here and remove the injection?